src/chunk.rs

```diff
@@ -0,0 +1,43 @@
+use bevy::math::Vec3;
+
+pub const CHUNK_SIZE: i32 = 32;
+pub const PADDED_CHUNK_SIZE: i32 = CHUNK_SIZE + 2;
+pub const PADDED_CHUNK_USIZE: usize = PADDED_CHUNK_SIZE as usize;
+pub const CHUNK_LENGTH: usize =
+    (PADDED_CHUNK_SIZE * PADDED_CHUNK_SIZE * PADDED_CHUNK_SIZE) as usize;
```

Comment:
Mixing `i32` and `usize` smells a bit. Generally sizes are unsigned and thus `usize` is a good fit.

Reply:
I agree.